.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -17,6 +17,9 @@
 
 **Which issue(s) this PR fixes**:
 <!-- Add a link to the JIRA issue. Otherwise, put "no issue." -->
+```
+jql=key in (D2IQ-<NUMBER>)
+```
```

Comment:
```suggestion
<!-- Add a link to the JIRA issue. Otherwise, put "no issue." -->
* jql=key in (D2IQ-<NUMBER>)
* https://jira.d2iq.com/browse/D2iQ-<NUMBER>
-->
```